promscale/installation/recomm-guide.md

```diff
@@ -0,0 +1,40 @@
+# Resource recommendation guide
+
+The resource recommendation guide is to help you understand the compute and disk requirements for Promscale based on your ingest rate and retention. 
+
+## Metrics
+
+### Prometheus remote write
+
+Below are **compute recommendations** for the Promscale connector and TimescaleDB:
+
+| Ingestion Rate          | Connector CPU      | Connector Memory   | DB CPU | DB Memory| DB connections |
+| --------                | -------- | -------- | -------|----------|----------------|
+| 10k samples/sec      |     0.5     |      250 MB    |    1    |    4 GB      |          1      |
+| 50k samples/sec      |     2     |     700 MB     |    4    |     16 GB     |       2         |
+| 100k samples/sec   |    4      |     2 GB     |          8     |       32 GB         |  4|
+| 200k samples/sec   |     8     |     4.5 GB     |     16   |      64 GB    |        8        |
+| 500k samples/sec   |    32      |     9 GB     |     32   |     128 GB    |        64        |
+
+Below are **disk recommendations** for the TimescaleDB:
```

Comment:
```suggestion
Disk recommendations for the TimescaleDB are:
```
https://developers.google.com/style/word-list?hl=en does not recommend using below for the position in the document.